### PR TITLE
Fixes for whisper support

### PR DESCRIPTION
**Pull request by lemieux**
The conversation was wrongly marked at initiated when it was not.

### CODE REVIEW

File: src/js/controllers/chatController.js [hunk 1]
```diff
@@ -153,7 +153,7 @@ module.exports = ViewController.extend({
                 .then(this._getConversation)
                 .then(this._initFaye)
                 .then(_.bind(function(conversation) {
-                    this.conversationInitiated = true;
+                    this.conversationInitiated = !conversation.isNew();
```
Comment (juliangarritano) [inline src/js/controllers/chatController.js]: So what does this fix, exactly?
We will get whispers from faye, now?

Comment (lemieux) [inline src/js/controllers/chatController.js]: Yeah, in this flow, the conversation that you get from `_getConversation` can be a new one (hence `!conversation.isNew()`) if no conversation was started. `_initFaye` is a noop in that case.

Later when you track something, it rechecks the server to see if a conversation was created and if it is the case, it connects to faye. It also fetches the conversation and the messages, so the first whispers might come throught the `fetch` call but everything else will go through faye.

Comment (lemieux) [inline src/js/controllers/chatController.js]: That's why it was only working on reload... the conversation was always marked as initiated so it was not refetching from the server.

**Comment by juliangarritano** [comment]
:+1:

**Comment by mspensieri** [comment]
:+1:

**Pull request merged**